### ISSUE TITLE
Unexpected background, pointer and missing focus indicator of disabled group buttons, when in ToolBar's overflow dropdown

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-themes#199]
- [ ] All SASS themes - disabled group button has pointer cursor on hover, instead of default cursor:
  - Use the following Dojo - http://dojo.telerik.com/AdiXoLIh
  - Click on the overflow button
  - Hover Bold - it has pointer
  - Click on Bold - its text and icon change color

- [ ] Bootstrap - unexpected background of disabled group button:
  - Use the following Dojo - http://dojo.telerik.com/UwaVaQaJ
  - Click on the overflow button
  - Bold has background
![image](https://user-images.githubusercontent.com/6614183/40611522-4d188fb2-627f-11e8-927a-5e0cfc3561ce.png)

- [x] Material - no focus indicator of disabled group button:
  - Use the setup from the following Dojo - http://dojo.telerik.com/UwaVaQaJ with Material theme
  - Click on the overflow button - Action button is focused
  - Press Down arrow - Bold is focused, but there is no visible indication for that

**Comment by inikolova** [comment]
It appears the issue with with Material theme has been already addressed -> http://dojo.telerik.com/UwaVaQaJ/3 , https://stackblitz.com/edit/angular-kkzexb?file=index.html
